app/component/itinerary/ItineraryPage.js

```diff
@@ -1238,6 +1238,7 @@ export default function ItineraryPage(props, context) {
             mapLayerRef={mapLayerRef}
             isNavigatorIntroDismissed={isNavigatorIntroDismissed}
             updateLegs={updateStoredItinerary}
+            forceStartAt={storedItinerary?.params?.forceStartAt || undefined}
           />
```

Comment:
`|| undefined` seems unnecessary as that's what is evaluated when forcestart is not set